apps/trustlab/src/payload/blocks/PageHeader.js

```diff
@@ -0,0 +1,61 @@
+import { image, richText } from "@commons-ui/payload";
+import { ColourTextField } from "@nouance/payload-better-fields-plugin/ColourText";
+import { validateHTMLColorHex } from "validate-color";
+
+const PageHeader = {
+  slug: "page-header",
+  imageURL: "/images/cms/blocks/page-header.png",
+  fields: [
+    {
+      type: "text",
+      name: "title",
+      required: true,
+    },
+    richText({
+      name: "description",
+      localized: true,
+    }),
+    image({
+      overrides: {
+        name: "image",
+      },
+    }),
+    {
+      type: "checkbox",
+      name: "isPageHeader",
+      defaultValue: true,
+      label: "Is Page Header",
```

Comment:
1. Why do we need `label` if it's `hidden`?
2. Since even `readOnly` fields can be edited via APIs, why not add `isPageHeader` using `afterRead` hook?